.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.7
```

Comment:
is this actually what this project was using before? i wonder if we can get it onto 3.1.3